src/app/lib/button/button-demo/button-demo.component.html

```diff
@@ -7,17 +7,17 @@ <h5>
     </h5>
     <div class="sample-buttons">
         <button hc-button title="Primary Button" color="primary">Primary</button>
-        <button hc-button title="Primary-Alt1 Button" color="primary-alt1">Primary Alt1</button>
-        <button hc-button title="Primary-Alt2 Button" color="primary-alt2">Primary Alt2</button>
-        <button hc-button title="Primary-Alt3 Button" color="primary-alt3" >Primary Alt3</button>
+        <button hc-button title="Primary-Alt Button" color="primary-alt">Primary Alt</button>
```

Comment:
I wonder if there's a similarly descriptive name to replace `Primary-Alt`

Reply:
Yeah, there may be. Any suggestions? I think there's likely another round of refactoring to come for this component soon, so I'll likely leave as is for now.